query/src/test/java/org/infinispan/query/api/PutAllTest.java

```diff
@@ -159,6 +160,21 @@ public void testOverwriteIndexedValue() {
       StaticTestingErrorHandler.assertAllGood(cache);
    }
 
+   @Test
+   public void testOverwriteIndexedValue_heavyLoad() {
```

Comment:
I didn't check this locally, so I'll trust that you checked this failed before your patch.

Reply:
> LGTM, see comments below.

Thanks for the review @yrodiere!

Reply:
actually it does not fail, it hangs